lib/ApiInterface.py

```diff
@@ -156,3 +156,10 @@ def get_activity_session(self, activity, session):
         except:
             raise
 
+    def get_workflow(self,workflow):
+        api_path = f"/workflows/{workflow}"
+        try:
+            api_response = self.request("GET", api_path)
+            return api_response
+        except:
```

Comment:
Generally it's not a good idea to have "except" without a exception type

Reply:
since none of the functions in this file have an exception type, I did not add here as well